grails-app/services/org/pih/warehouse/shipping/ShipmentService.groovy

```diff
@@ -699,26 +699,15 @@ class ShipmentService {
         }
     }
 
-    /**
-     *
-     * @param shipmentItem
-     * @return
-     */
-    boolean validateShipmentItem(ShipmentItem shipmentItem) {
-        boolean binLocationRequired = (shipmentItem.binLocation ?: false)
-        return validateShipmentItem(shipmentItem, binLocationRequired)
-    }
-
     /**
      * Validate the shipment item when it's being added to the shipment.
      *
      * @param shipmentItem shipment item to validate
-     * @param binLocationRequired if shipment item has a bin location we validate quantity only against that bin
      * @return
      */
-    boolean validateShipmentItem(ShipmentItem shipmentItem, boolean binLocationRequired) {
```

Comment:
binLocationRequired wasn't actually being used (looks like it was [leftover after an old refactor](https://github.com/openboxes/openboxes/commit/74b5206d991bbde9276fce60c2d732704f56aa91#diff-a701539d6f9ea99ef6718917e53ab215d7b15ac68ecd7a34443ff2a689f8e823)) so I merged the above method with this one.